conflation/trace_fetching/mapillary.py

```diff
@@ -20,11 +21,19 @@
 SKIP_IF_FEWER_IMAGES_THAN_DEFAULT = (
     10  # We will skip any sequences if they have fewer than this number of images
 )
+
+TILES_URL = (
+    "https://tiles.mapillary.com/maps/vtp/mly_map_feature_point/2/{}/{}/{}?access_token={}"
+)
+
 SEQUENCE_URL = "https://a.mapillary.com/v3/sequences_without_images?client_id={}&bbox={}&per_page={}&start_date={}"
 IMAGES_URL = "https://a.mapillary.com/v3/images?client_id={}&sequence_keys={}&per_page={}"
+MAX_FILES_IN_DIR = 500  # Maximum number of files we will put in one directory
 
 
-def run(bbox: str, traces_dir: str, tmp_dir: str, config: dict, processes: int) -> int:
+def run(
```

Comment:
The changes in this file represent changes I made for v4. Really just adding the `access_token` and some stuff related to changing lon,lat into tiles.